bitwarden_license/src/app/app.component.ts

```diff
@@ -14,6 +15,7 @@ export class AppComponent extends BaseAppComponent {
 
         this.policyListService.addPolicies([
             new MaximumVaultTimeoutPolicy(),
+            new DisablePersonalVaultExportPolicy(),
```

Comment:
I'm sure there is a reason for this, but why are these the only two policies declared here?

Reply:
The other policies are defined in the OSS AppComponent.